ansible/README.md

```diff
@@ -0,0 +1,178 @@
+    Encryption successful
+```
+
+Then, when you run the playbook, you'll use the `--ask-vault-pass` flag.
+
+    $ ansible-playbook mediawiki-install.yml --ask-vault-pass
+    Vault password:
+
+If you want to run the command in a script, you can also pass the vault
+password in a file.
+
+## Server configuration
+
+We include an `mwiki-default.conf.tmpl` in this directory.  You will need to copy that to your .conf file like so:
+
+    $ cp mwiki-default.conf.tmpl mwiki-default.conf
+    # Edit ServerName to be whatever you want to call your host
+
+If you know what changes you want to make to your Apache configuration
+(e.g. adding SSL), you can make those here.  You will at least need
+to fill in the Server Name.
+
+## Mediawiki Farm (i.e., multiple wikis)
+
+Our configuration for multiple wikis is currently closely based on
+[documentation in another
+branch](https://github.com/OpenTechStrategies/librehq-wikis/blob/mediawiki-updates/MEDIAWIKI_INSTALL.md)
+of this repository.  We expect these to be united on the master branch
+in the future.
+
+To use the mediawiki farm parts of this Ansible playbook, you will need
+to add the database password to `test1.yml` and `test2.yml`.
+
+    $ cp mediawikifarmdocs/test1.yml.tmpl mediawikifarmdocs/test1.yml
+    # Edit database password to match the one you entered in your vault
+
+
+## Local testing
+
+Locally, Ansible works well with Vagrant.
+
+    $ sudo apt-get install vagrant
+    $ sudo apt-get install virtualbox
+
+Note that the [Vagrant
+documentation](https://www.vagrantup.com/docs/installation/) suggests
+_not_ using the version of Vagrant packaged by your operating system,
+but rather using the more complete and up-to-date versions they
+[provider](https://www.vagrantup.com/downloads.html). Either way,
+downloading may take a few minutes.  If you choose to download the
+`.deb` file, you can install it with `sudo dpkg -i
+/path/to/vagrant_2.2.1_x86_64.deb`. Test your local installation with
+the following commands:
+
+    # Adds a Debian image to your VirtualBox installation
+    $ vagrant box add debian/testing64
+
+    # Since there is a committed Vagrantfile in this repo, you don't
+    # need to `vagrant init` and can instead move on to these commands:
+    
+    $ emacs Vagrantfile
+      # Add: config.vm.box = "debian/testing64"
+
+    $ vagrant up
+
+You may get a persistent error like `default: Warning: Authentication
+failure. Retrying...`.  Check that the virtual machine is being created
+correctly in VirtualBox (you can look in the GUI to see if it appears
+and is running).  The quickest and simplest way to resolve this is by
+copying [the Vagrant default key
+files](https://github.com/hashicorp/vagrant/tree/master/keys) to your
+`.ssh` directory (on your host machine, not the new virtual guest
+machine).  The Vagrantfile assumes that this default private key is
+present at `~/.ssh/vagrant-insecure`, as noted in the line
+`config.ssh.private_key_path = "~/.ssh/vagrant-insecure"`.
```

Comment:
I got this error and had to use these default key files.  Also, I had to add a newline to those files to get past the problem.  See [this bug report](https://github.com/hashicorp/vagrant/issues/10333).  So we should add that to the instructions.